DataFormats/L1TMuon/interface/EMTFShower.h

```diff
@@ -0,0 +1,32 @@
+#ifndef DataFormats_L1TMuon_EMTFShower_h
+#define DataFormats_L1TMuon_EMTFShower_h
+
+namespace l1t {
+
+  class EMTFShower {
```

Comment:
Right, ultimately, it all comes together into the uGT. So perhaps like you suggest something more general. I can change it to ```l1t::Shower``` or ```l1t::RegionalShower``` (since it's per sector). Not sure if we would want to call it ```l1t::RegionalMuonShower``` since we're trying to capture hadronic showers and similar things, but that don't look at all like muons.